ui/app/pages/connected-sites/connected-sites.container.js

```diff
@@ -0,0 +1,41 @@
+import { connect } from 'react-redux'
+import ConnectedSites from './connected-sites.component'
+import {
+  getFirstPermissionRequest,
+  getNativeCurrency,
+  getAccountsWithLabels,
+} from '../../selectors/selectors'
+import { approvePermissionsRequest, rejectPermissionsRequest, showModal } from '../../store/actions'
+
+const mapStateToProps = state => {
+  const permissionsRequest = getFirstPermissionRequest(state)
+  const { metadata = {} } = permissionsRequest || {}
+  const { origin } = metadata
+  const nativeCurrency = getNativeCurrency(state)
+
+  const accountsWithLabels = getAccountsWithLabels(state)
+
+  return {
+    permissionsRequest,
```

Comment:
These props and methods all appear to be unused? I'm guessing they were moved into the `ConnectedSitesList` component and left here accidentally.

Reply:
ab0d66fa1